make-rules/x11.mk

```diff
@@ -0,0 +1,90 @@
+#
+# CDDL HEADER START
+#
+# The contents of this file are subject to the terms of the
+# Common Development and Distribution License (the "License").
+# You may not use this file except in compliance with the License.
+#
+# You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
+# or http://www.opensolaris.org/os/licensing.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+#
+# When distributing Covered Code, include this CDDL HEADER in each
+# file and include the License file at usr/src/OPENSOLARIS.LICENSE.
+# If applicable, add the following below this CDDL HEADER, with the
+# fields enclosed by brackets "[]" replaced with your own identifying
+# information: Portions Copyright [yyyy] [name of copyright owner]
+#
+# CDDL HEADER END
+#
+# Copyright (c) 2017, Aurelien Larcher. All rights reserved.
+#
+
+# All X11 components should now prefer 64-bits unless specified
+
+PREFERRED_BITS=64
+
+#
+# Provide macros for commonly used urls
+#
+
+XORG_PROJECT_URL = https://www.x.org/
+XORG_BASE_URL    = https://www.x.org/releases/individual
+SOLARIS_XORG_URL = https://github.com/oracle/solaris-userland
+
+XORG_APP_BASE_URL     = $(XORG_BASE_URL)/app
+XORG_DATA_BASE_URL    = $(XORG_BASE_URL)/data
+XORG_DOC_BASE_URL     = $(XORG_BASE_URL)/doc
+XORG_DRIVER_BASE_URL  = $(XORG_BASE_URL)/driver
+XORG_FONT_BASE_URL    = $(XORG_BASE_URL)/font
+XORG_LIB_BASE_URL     = $(XORG_BASE_URL)/lib
+XORG_PROTO_BASE_URL   = $(XORG_BASE_URL)/proto
+XORG_TEST_BASE_URL    = $(XORG_BASE_URL)/test
+XORG_UTIL_BASE_URL    = $(XORG_BASE_URL)/util
+XORG_XCB_BASE_URL     = $(XORG_BASE_URL)/xcb
+XORG_XSERVER_BASE_URL = $(XORG_BASE_URL)/xserver
+
+#
+# Define library install paths
+#
+
+XORG_LIBDIR.32 = $(USRLIBDIR)/xorg	
+XORG_LIBDIR.64 = $(USRLIBDIR)/xorg/$(MACH64)	
+XORG_LIBDIR    = $(XORG_LIBDIR.$(BITS))
+
+#
+# Common macros for font packages
+#
```

Comment:
Sorry, I don't see their usage in our xorg components. Do we really need them now?

Reply:
The corresponding components is in another branch.